tests/integration/test_global.py

```diff
@@ -726,3 +726,24 @@ def test_global_uninstall_only_reverts_failing(
     assert not tmp_path.joinpath("envs", "dummy-a").is_dir()
     assert dummy_b.is_file()
     assert tmp_path.joinpath("envs", "dummy-b").is_dir()
+
+
+def test_auto_self_expose(pixi: Path, tmp_path: Path, test_data: Path) -> None:
+    env = {"PIXI_HOME": str(tmp_path)}
+    non_self_expose_channel = test_data.joinpath("non_self_expose_channel/output").as_uri()
```

Comment:
Let's also make a fixture for that one for consistency

Reply:
Fair enough